sevntu-checks/src/main/resources/com/github/sevntu/checkstyle/checks/coding/messages.properties

```diff
@@ -19,6 +19,7 @@ finalize.implementation.public = finalize() method should have a "protected" vis
 finalize.implementation.useless = finalize() method is useless: it does nothing except for calling super.finalize().
 finalize.implementation.missed.super.finalize = You have to call super.finalize() right after finally opening brace.
 forbid.certain.imports=Import ''{1}'' should not match ''{0}'' pattern, it is forbidden.
+forbid.certain.method=Call to ''{0}'' method (matches pattern {1}) with ''{2}'' arguments (matches pattern {3}) is forbidden.
```

Comment:
How come 0 and 2 are wrapped with quotes, but 1 and 3 aren't?
3 could definitely contain spaces.

Reply:
The resulting message looks like this:
Call to 'assertFalse' method (matches pattern ^assert(True|False)$) with '1' arguments (matches pattern 1) is forbidden.

Reply:
This doesn't answer my question on why they aren't wrapped in quotes.

> matches pattern ^assert(True|False)$) 

Makes it hard to understand where the text ends and the expression begins.

Reply:
Done. I have added quotes around {1} and {3}.